client/signup/steps/theme-selection/index.jsx

```diff
@@ -101,42 +122,31 @@ class ThemeSelectionStep extends Component {
 
 	render = () => {
 		const defaultDependencies = this.props.useHeadstart ? { themeSlugWithRepo: 'pub/twentysixteen' } : undefined;
+		const { translate } = this.props;
 
-		const pressableWrapperClassName = classNames( {
-			'theme-selection__pressable-wrapper': true,
-			'is-hidden': ! this.state.showPressable,
-		} );
-
-		const themesWrapperClassName = classNames( {
-			'theme-selection__themes-wrapper': true,
-			'is-hidden': this.state.showPressable,
-		} );
+		const headerText = this.state.showPressable
+			? translate( 'Upload your WordPress Theme' )
+			: translate( 'Choose a theme.' );
 
-		const { translate } = this.props;
+		const subHeaderText = this.state.showPressable
+			? translate( 'Our partner Pressable is here to help you', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Our partner Pressable is here to help you' )` *ES Score: 13.91*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/signup-themes-step-pressable-overlay-no-scroll)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).